api/handlers_test.go

```diff
@@ -431,27 +438,34 @@ func TestPulses_HappyPath(t *testing.T) {
 func TestPulses_PulsesWithRecords(t *testing.T) {
 	defer testutils.TruncateTables(t, testDB, []interface{}{models.Record{}, models.JetDrop{}, models.Pulse{}})
 
+	s := storage.NewStorage(testDB)
+	fnCreateData := func(t *testing.T, recordAmount int, pulse models.Pulse) models.JetDrop {
```

Comment:
This function looks same as previous, just with returning of JetDrop. Maybe move it to `testutils` and use in both places? (just ignore returning value in `TestPulse_PulseWithRecords` test cause we don't need it there)